src/types/TaxForm.ts

```diff
@@ -5,6 +5,10 @@ export interface TaxForm extends Required<TaxFormUserInputBase<number>> {
   t1r10_vydavky: number;
   t1r2_prijmy: number;
   t1r10_prijmy: number;
+  /**   04 - Priezvisko*/
+  r004_priezvisko: string;
```

Comment:
Toto treba pridat tu? Ak to je v TaxUserInput tak by to nemalo byt potrebne aj tu. Teoreticky ale mozeme tieto dva typy uplne separovat za cenu nejakej duplicity, lebo asi to zacina robit problemy ze su tak prepojene.

Reply:
Z `TaxFormUserInputBase` som odstranil polozky pre meno a priezvisko a namiesto toho som spravil jedno spojene `meno_priezvisko`. Kedze samostatne meno a priezvisko treba do DP tak som to presunl sem to `TaxForm`. [V metode `calculate` sa to parsuje](https://github.com/wormyy/priznanie-digital/pull/65/files#diff-8b4efb9c6b5cef1d553b17eab5ce7d07).

Reply:
okej, to potom sedi 👍